dist/index.html

```diff
@@ -140,6 +144,21 @@ <h1> Our recipes </h1>
       </section>
       <article class="recipe-grid">
         <div class="col">
+          <article class="individual-recipe-container">
+            <section class="add-panel">
+              <div class="plus-symbol">+</div>
+              <h4> Add to recipes to cook</h4>
+            </section>
```

Comment:
I don't think we need this here since we're creating these elements in `domUpdates`?

Reply:
removed